calculator.js

```diff
@@ -0,0 +1,85 @@
+(function() {
+    "use strict";
+    var result = 0;
+
+    var buttons = document.querySelector('.allButtons');
+    var numberButtons = buttons.querySelectorAll('.number');
+
+    function Screen() {
```

Comment:
Now move Screen class to a dedicated JS file. As number of classes (modules) grows, we do not have the luxury to keep them in a single file. The general rule of thumb is to have one script file per class.